charts/openshift-metering/values.yaml

```diff
@@ -544,6 +544,16 @@ presto:
             useInstanceCredentials: null
 
         extraConnectorFiles: []
+        prometheus:
+          enabled: true
```

Comment:
Hmm the more I think about this, I feel that we should disable this connector by default until the connector is a supported configuration option. This would also make it easier to mock in the e2e suite as we'd be explicitly enabling the connector.

Reply:
I think it's easier to test initial functionality, like we're doing right now, if it's supposed to be enabled and working by default. Also the connector is the only way to get info from Prometheus, so `not enabled` is exactly the same as `not present`, so I'm not sure I see the benefits for mocking?

Reply:
Okay, that's understandable, and I realize that there are many steps to actually enable the workflow, and this PR just exposes another connector to the metering stack. With that said, I think we can keep enabling it by default then.